test/checks/tables/th-has-data-cells.js

```diff
@@ -24,6 +24,8 @@ describe('th-has-data-cells', function () {
 			'<table>' +
 			'  <tr> <th>hi</th> <td>hello</td> </tr>' +
 			'  <tr> <th>hi</th> <td>hello</td> </tr>' +
+			'  <tr> <td>hello</td> <th>hi</th> </tr>' +
```

Comment:
Last time I checked, this technique did not work on all major screen reader - browser combinations.

What screen readers have you tested?